src/commands/fun/poll.ts

```diff
@@ -25,7 +33,8 @@ export default class extends Command {
 
 		const pollEmbed = new MessageEmbed()
 			.setTitle(`Poll from ${msg.member.displayName}`)
-			.addField(question, choiceText)
+			.addField('Question', question)
+			.addField('Choices', choiceText, false)
```

Comment:
Is this intended to separate the question and choice fields completely?

Reply:
Yeah, my reasoning behind this is that questions beyond 256 characters were throwing errors. This PR adds length checking for questions and increases the question char limit to 1024 instead of 256. A field can't have a title of more than 256 and a body more than 1024.